<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="HL_Experiment">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; text-decoration: underline;&quot;&gt;Instructions:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;1. Press stimulation button &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;2. Wait until stimulation is finished&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;3. Answer the questions bellow&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;4. Repeat until the experiment is finished &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HL_questions">
        <property name="topMargin">
         <number>25</number>
        </property>
        <property name="bottomMargin">
         <number>25</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="VL_questions">
          <property name="spacing">
           <number>25</number>
          </property>
          <item>
           <widget class="QLabel" name="L_1">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Do you feel something ?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="L_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Where do you feel it ?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="L_3">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>What is the pain level ?
(0 = no pain, 10 = worst imaginable pain) </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>25</number>
          </property>
          <item>
           <widget class="QComboBox" name="CB_1">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Yes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>No</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="CB_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Referred area</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Near electrodes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Both</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SB_3">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="bottomMargin">
         <number>40</number>
        </property>
        <item alignment="Qt::AlignTop">
         <widget class="QLabel" name="L_comment">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Comments: (Optional)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignRight|Qt::AlignBottom">
         <widget class="QLabel" name="L_stimulationON">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Stimulation ON, wait </string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QLabel" name="L_LED">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>20</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>green_circle.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="PB_STIMULATE">
        <property name="text">
         <string>Next Stimulation &gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
